### PR TITLE
Fixed #4 and #5

### LINKED ISSUE #4: Profile/ProfileExperienceItem
# End Result:
image

# Description:
A simple item that displays the experience data in text. 

# props:
* data - (object) shape: {title: string, role: string, director: string, company: string, location: string}
* style - (object) applied to root element

### PR DESCRIPTION
**Pull request by monaldev**
I have added two components ProfileExperienceCategory and ProfileExperienceItem. 

Please review my changes and let me know your feedback

### CODE REVIEW

File: src/Profile/components/profile_experience_item.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import {
+  CardHeader,
+} from 'material-ui';
+
+const ProfileExperienceItem = ({ data, style }) => (
+  <div style={style}>
+    <CardHeader title="TITLE:" subtitle={data.title} />
```
Comment (rohit-ravikoti) [inline src/Profile/components/profile_experience_item.js]: Please switch title and subtitle for each of these.

File: src/Profile/stories/profile_experience_category.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+          width: '350px',
+          margin: 'auto',
+          position: 'relative',
+        }}
+      >
+        {story()}
+      </div>
+    </LoadMuiTheme>
+  ))
+  .add('individual', () => {
```
Comment (rohit-ravikoti) [inline src/Profile/stories/profile_experience_category.js]: Please add another story with 'multiple'

File: src/Profile/stories/profile_experience_category.js [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+    const children = [
+      {
+        title: 'The lion king',
+        role: 'Ensenble/Mufasa understudy',
+        director: 'Julie Peloni',
+        company: 'Disney Entertainment',
+        location: 'Los Angeles',
+      },
+    ];
+    const count = children.length;
```
Comment (rohit-ravikoti) [inline src/Profile/stories/profile_experience_category.js]: counting should be done in ProfileExperienceCategory item itself

File: src/Profile/components/profile_experience_category.js [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+
+const ProfileExperienceCategory = ({
+  children,
+  categoryName,
+}) => (
+  <Card
+    initiallyExpanded
+  >
+    <CardTitle
+      title={categoryName}
```
Comment (rohit-ravikoti) [inline src/Profile/components/profile_experience_category.js]: append count of children to categoryName

File: src/Profile/components/profile_experience_item.js (hunk 1 above)
Comment (monaldev) [inline src/Profile/components/profile_experience_item.js]: I have switched Title and Subtitle

File: src/Profile/stories/profile_experience_category.js (hunk 2 above)
Comment (monaldev) [inline src/Profile/stories/profile_experience_category.js]: I have added new story with 'multiple'

File: src/Profile/stories/profile_experience_category.js (hunk 3 above)
Comment (monaldev) [inline src/Profile/stories/profile_experience_category.js]: Counting Moved to ProfileExperienceCategory

File: src/Profile/components/profile_experience_category.js (hunk 4 above)
Comment (monaldev) [inline src/Profile/components/profile_experience_category.js]: I have appended count to children categoryName

**Pull request merged**